Src/FluentAssertions/Specialized/ExceptionAssertions.cs

```diff
@@ -266,12 +266,7 @@ private static string BuildExceptionsString(IEnumerable<TException> exceptions)
 
     private sealed class ExceptionMessageAssertion
     {
-        public ExceptionMessageAssertion()
-        {
-            Context = "exception message";
-        }
-
-        public string Context { get; }
+        private readonly string context = "exception message";
```

Comment:
Could also be `const`, that will make the lambda expression where it's used non-capturing, which enables the compiler to statically cache the lambda.

Reply:
> Convert local variable or field into constant (private accessibility)

Qodana thinks the same :D